include/deal.II/grid/grid_refinement.h

```diff
@@ -96,31 +96,29 @@ namespace GridRefinement
    *
    * <li> Sort the cells according to descending values of @p criteria.
    *
-   * <li> Set the refinement threshold to be the criterion belonging to the
-   * cell at position @p top_fraction_of_cells times
-   * Triangulation::n_active_cells().
+   * <li> Mark the @p top_fraction_of_cells times
+   * Triangulation::n_active_cells() cells with the largest refinement criteria
```

Comment:
Yes, I guess I could say "...active cells with...".

What place above are you referring to?